pages/api/storeNlpData.ts

```diff
@@ -0,0 +1,38 @@
+import { MongoClient } from "mongodb"
+import { NextApiRequest, NextApiResponse } from "next"
+import redis from "@/utils/redis"
+import * as keys from "@/utils/keys"
+import { NLP_KEY } from "@/constants"
+
+export default async function handler(_req: NextApiRequest, res: NextApiResponse) {
+  let numberOfDocuments = 0
+  const date = new Date().toLocaleDateString()
+  await MongoClient.connect(
+    `mongodb://${keys.mongoUser}:${keys.mongoPassword}@${keys.serverHost}:${keys.mongoPort}`,
+    {}
+  ).then(async (client) => {
+    await redis.keys(`${NLP_KEY}*`).then(
+      async (keys) =>
+        await redis.mget(keys).then(async (values) => {
```

Comment:
redis.mget means "get multiple" and takes an array, so we can just pass all the keys from the previous request into mget